autobot/cli.py

```diff
@@ -10,61 +10,63 @@
 """CLI commands."""
 
 import sys
+import os
 
 import click
-import yaml
 from github3 import login, repository
 
 from autobot.api import BotAPI
 from autobot.config_loader import Config
 
+dotenv_path = os.path.join(os.path.abspath(os.path.join(__file__, "..")), ".env")
+
+ini_path = os.path.join(os.path.abspath(os.path.join(__file__, "..")), "config.ini")
+
 
 @click.group(context_settings=dict(help_option_names=["-h", "--help"]))
 def main():
-    """Autobot cli."""
+    """Autobot CLI."""
     pass
 
 
 @click.group(context_settings=dict(help_option_names=["-h", "--help"]))
 def report():
-    """Autobot report cli."""
+    """Autobot report CLI."""
     pass
 
 
 @click.command(context_settings=dict(help_option_names=["-h", "--help"]))
 @click.option(
     "--owner",
     default="inveniosoftware",
-    help="The repo owner we plan to offer the service to.",
+    help="The repo owner/organization the service is offered to.",
 )
 @click.option(
     "--repo", multiple=True, help="The repositories to check for notifications."
 )
 @click.option("--maintainer", multiple=True, help="The maintainers to notify.")
-@click.option("--format", default="json", help="The result format.")
+@click.option("--format", default="json", help="The report format.")
 def show(owner, repo, maintainer, format):
-    """Autobot report show cli."""
-    conf = Config(
+    """Autobot report show CLI."""
+    conf = Config.load(
         AUTOBOT_OWNER=owner,
         AUTOBOT_REPOS=[r for r in repo],
         AUTOBOT_MAINTAINERS=[m for m in maintainer],
+        env=dotenv_path,
+        ini=ini_path,
```

Comment:
`ini_path` can be a `--config` option for all the CLI commands. For adding it to all the commands you can take some inspiration from [this example](https://github.com/inveniosoftware/metainvenio/blob/20eb3e8ac7c0d6db6b515c6a1f1e0b158d7fd4fe/metainvenio/cli/main.py#L36-L51)